src/algebra/char_p/exp_char.lean

```diff
@@ -0,0 +1,132 @@
+/-
+Copyright (c) 2021 Jakob Scholbach. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Author: Jakob Scholbach.
```

Comment:
```suggestion
Authors: Jakob Scholbach
```

Jakob, I think that the linter is complaining about your new file and, again I think, that this suggestion should fix the linter issue.